Dockerfile

```diff
@@ -13,28 +13,31 @@ RUN go get github.com/tools/godep
 RUN echo "Installing the go-bindata tool"
 RUN go get github.com/jteeuwen/go-bindata/...
 
-ADD . /src/github.com/
+ADD . /src/github.com/openwhisk/openwhisk-cli
+
+#RUN git clone http://github.com/openwhisk/openwhisk-cli /src/github.com/openwhisk/openwhisk-cli
+
+#RUN cd /src/github.com/openwhisk/openwhisk-cli && git checkout master
 
 # Load all of the dependencies from the previously generated/saved godep generated godeps.json file
 RUN echo "Restoring Go dependencies"
-RUN cd /src/github.com/go-whisk-cli && /bin/godep restore -v
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/godep restore -v
 
 # Collect all translated strings into single .go module
 RUN echo "Generating i18n Go module"
-RUN cd /src/github.com/go-whisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
-RUN cd /src/github.com/go-whisk && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
```

Comment:
I do not think we need the code from Line 27 to Line 28. We used to be lack of the Go modules, because they are not in the go project structure under $gopath. 

Since we restore to put the code under a go project structure, the i18n module should be found directly without the generation.